depends/libhdfs3/src/client/EncryptionZoneInfo.h

```diff
@@ -0,0 +1,84 @@
+/********************************************************************
```

Comment:
suggest a new file do not include line:1 ~ line:4 
standard apache source file should start from line:5